<launch>
    <!-- Start the low-level driver manager with parameters -->
    <node name="ax12_driver_manager" pkg="ax12_driver_core" type="manager.py" output="screen" required="true">
        <param name="port_name" type="str" value="/dev/ttyUSB0"/>
        <param name="baud_rate" type="int" value="1000000"/>
        <param name="max_motor_id" type="int" value="25"/>
        <param name="update_rate" type="int" value="1"/>
    </node>

    <!-- Load head parameters from YAML file -->
    <rosparam file="$(find wubble_camera_pan_tilt_controller)/config/wubble_head.yaml" command="load"/>

    <!-- Start the smart_arm driver -->
    <node name="head_spawner" pkg="ax12_driver_core" type="driver_ctrl.py"
          args="start $(find wubble_camera_pan_tilt_controller)/src/wubble_camera_pan_tilt_controller/camera_pan_tilt_ax12.py"
          output="screen"/>
</launch>
